widget/index.jsx

```diff
@@ -86,10 +86,17 @@ const config = {
   },
 };
 
+const InterCSS = fetch(
```

Comment:
This fetch being in our root makes the whole page reload on navigation. We'll need to incorporate it into the layout or find another solution. 

But for now; I propose not importing a font, just use web fonts